### DISCUSSION TITLE
Inaccurate program description in some places

### DISCUSSION THREAD

**Discussion by danfe** [akiraux/Akira#634] [General]
In several files, it describes itself as *The Linux Design Tool* which is ambiguous (it could be perceived as a tool which helps to design Linux kernel) and actually wrong (as it builds and works fine on e.g. FreeBSD).  Consider applying the following change:
```shell
$ sed -i.bak -e 's,The Linux,The UI/UX,' \
    README.md \
    data/com.github.akiraux.akira.appdata.xml.in.in \
    data/com.github.akiraux.akira.desktop.in.in \
    po/*.po \
    src/Dialogs/SettingsDialog.vala
```

**Comment by albfan** [comment]
One of the main goals for the project is to provide a native design tool on Linux, opposed to other design tools only available on Mac and Windows, I think that's the purpose of that definition.

Apart from the Linux vs GNU/Linux definition as an OS (which looks totally irrelevant for 99% of people) description looks pretty understandable, anyway, "The UI/UX Design Tool available on Linux" may be another option, to emphasize our Linux support objective.

About OS, we already built successfully on windows, and I suppose Mac is not there just because no one gave it a try. As open as we want to be, probably we design an app that can be used in any OS, and then your suggestion could make sense.

Suggestion: Probably just opening a PR, we can discuss there wether can or not the description, instead of showing a cryptic sed command. (Not saying is hard to read, just narrows the audience for your suggestion)

**Comment by danfe** [comment]
> One of the main goals for the project is to provide a native design tool on Linux, opposed to other design tools only available on Mac and Windows

While wanting to have such tool on GNU/Linux is a nice initial driver, sticking to a particular operating system in the long run and putting it onto the program description looks dubious.  There are lots of operating systems besides GNU/Linux which could very well benefit from UI/UX design tool.  Those Mac and Windows applications are not just unportable, they are also most likely proprietary software, and providing free (libre), open-source alternative would make a better, more honorable goal, appealing to a larger group of users as well.

> we already built successfully on windows, and I suppose Mac is not there just because no one gave it a try

Even less reasons to keep clinging to the L-word. :-)

> Probably just opening a PR

I'm not very comfortable with pull requests, they're quite confusing to me (just as Git itself), although I still might do that once we reach better consensus in this discussion.

> instead of showing a cryptic sed command.

Sapienti sat.

**Comment by Alecaddd** [comment]
Thanks for the suggestion, but I don't think I want to change the description.

The target platform of Akira and the core team is GNU/Linux and our target audience are Linux users.
The fact that Akira is available for Windows or FreeBSD is just a side effect of members of the community packaging the app for those platforms.

"The Linux Design Tool" is straightforward and I always like it.
I'm not against finding another description/payoff for Akira, but the reasons you list are not really valid to me.

I'll convert this into a discussion.
